src/GameEngine/GameEngine.cpp

```diff
@@ -258,12 +258,13 @@ namespace GameEngine {
             for(int player_index : *player_order){
                 // Player may have placed all his armies already
                 if(remaining[player_index] <= 0) continue;
+                Player::Player * current_player = players->at(player_index);
 
                 Terminal::print("");
-                Terminal::print("Now player #"+to_string(player_index+1)+" turns.");
+                Terminal::print("Player "+ current_player->get_color() + " turns.");
```

Comment:
I found this printout a little confusing in the console. 
For instance, I would personally replace it with one of the following:

> It is now the turn of Player [color].
>  It is currently Player [color]'s round.
>  It is now Player [color]'s turn.